modules/gltf/src/lib/extensions/KHR_texture_transform.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_texture_transform/README.md
+ */
+
+import {Vector3, Matrix3} from '@math.gl/core';
+import type {GLTF, GLTFMaterial, GLTFMeshPrimitive} from '../types/gltf-types';
+import type {GLTFLoaderOptions} from '../../gltf-loader';
+import {BYTES, COMPONENTS} from '../api/post-process-gltf';
```

Comment:
I fear this can become a circular import. Perhaps we can move those into a `gl-constants.ts` file?

In fact I think some of those are already defined in other places, e.g. loaders.gl/math module.

Reply:
Done